<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="signin">
       <widget class="QLabel" name="Header">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>英雄， 请登录您的账号</string>
        </property>
       </widget>
       <widget class="QPushButton" name="signinButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>190</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="gotoSignup">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>240</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>注册账号</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="userNameEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="userPwdEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>150</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="signup">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>欢迎客官报道</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="userNameEdit_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="userPwdEdit_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>重复密码</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="repeatPwdEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="signupButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>240</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
